### ISSUE TITLE
Top-down approach?

### ISSUE THREAD

**Issue by kbeckenrode** [waldronlab/MicrobiomeOntology#2]
Hi @wdduncan,

I'm using the top-down approach to develop the axioms between the most general classes. When I apply logic between two classes, I realize that I need to add the logic at a certain hierarchy, and not necessarily at its more general class. 

For example, I want to say "NCBITaxon:Bacteria" isapartof "OHMI:microbiome". "OHMI:microbiome" is a subclass of the more general term "entity". I don't want to say Bacteria isapartof entity, so that means I need to connect Bacteria (a general class) to microbiome. 

Can you verify my thinking here?

Thanks!

**Comment by wdduncan** [comment]
Yes. You are on the right track. You need to be careful in your use of `part of` vs. `has part' and quantifiers `some`, `only`.  

For example, the axiom:
```
NCBITaxon:Bacteria part of some OHMI:microbiome
```
says that **every** NCBITaxon:Bacteria is part of some OHMI:microbiome (which is false). But, this axiom:
```
OHMI:microbiome has part **some** NCBITaxon:Bacteria
```
says **every** OHMI:microbiome has part **NCBITaxon:Bacteria** (which may very well be true .. are there microbiomes that don't have bacteria?).

Does that help?

**Comment by lwaldron** [comment]
"Are there microbiomes that don't have bacteria" - probably not, but note that microbiomes also include viruses, eukaryotes, and protozoa, and some assays identify only one of those.

**Comment by wdduncan** [comment]
@lwaldron Thanks for the clarification!

@kbeckenrode Sounds like the axiom `OHMI:microbiome has part some NCBITaxon:Bacteria` is okay.  
If you are dealing with samples, you can use OBIB ( http://purl.obolibrary.org/obo/obib.owl) or OBI (http://purl.obolibrary.org/obo/obi.owl) to represent the specimen taken from the patient. See for example OBI's specimen from organism (http://purl.obolibrary.org/obo/OBI_0001479), or something more specific, such as saliva specimen (http://purl.obolibrary.org/obo/OBI_0002507).  

Using these classes, you distinguish between the microbiome, the specimen collected from the microbiome, and the organisms that were detected by assays.  

Here is figure using OBI relations you might find helpful: https://journals.plos.org/plosone/article/figure/image?size=large&id=10.1371/journal.pone.0154556.g002. Although, it might be more detailed than you need it.

**Comment by kbeckenrode** [comment]
@wdduncan I don't think we will include patient samples at this time. But, I wonder if "specimen from organism" would be alternative way to define 'body site' from UBERON. The OBI example you shared might be a more direct way of saying "Bacteria X is located in Y"

**Comment by wdduncan** [comment]
Yes. You can link the specimen to the site from which it was collected. So, could have something like:
```
Bacteria X located in some Specimen Y .
Specimen Y collect from Anatomical Site Z .
```
